server/middleware/setUpWebSecurity.ts

```diff
@@ -38,6 +38,7 @@ export default function setUpWebSecurity(): Router {
           fontSrc: ["'self'"],
         },
       },
+      crossOriginEmbedderPolicy: true,
```

Comment:
This sets the Cross-Origin-Embedder-Policy to require-corp which will only allow loading of resources from same origin, or resources explicitly marked as loadable from another origin using the crossorigin attribute.